main.go

```diff
@@ -152,9 +174,56 @@ func main() {
 				generateCmd.Env = append(generateCmd.Env,
 					fmt.Sprintf("VIRTINK_CONTROL_PLANE_MACHINE_ROOTFS_CDI_IMAGE=%s", controlPlaneMachineRootfsImage),
 					fmt.Sprintf("VIRTINK_WORKER_MACHINE_ROOTFS_CDI_IMAGE=%s", workerMachineRootfsImage),
-					fmt.Sprintf("VIRTINK_NODE_ADDRESSES=[%v]", strings.Join(machineAddresses, ",")),
+					fmt.Sprintf("VIRTINK_IP_POOL_NAME=%s", args[0]),
 				)
 
+				if err := runCommand(exec.Command("kubectl", "delete", "ippool.ipam.metal3.io", args[0], "--namespace", targetNamespace, "--wait", "--ignore-not-found")); err != nil {
+					return fmt.Errorf("delete IPPool: %s", err)
+				}
+
+				ipPool := &ipamv1.IPPool{
+					TypeMeta: metav1.TypeMeta{
```

Comment:
Why not use yaml template here?

Reply:
Use a ipPool object is easy to write, as we use `kubectl` directly, use a yaml template is more appropriate, I will fix it.